ui/panels-plugin/src/plugins/time-series-table/SeriesName.tsx

```diff
@@ -0,0 +1,92 @@
+// Copyright 2024 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Labels } from '@perses-dev/core';
+import { ReactElement, MouseEvent, ReactNode } from 'react';
+import { useSnackbar } from '@perses-dev/components';
+import { Typography } from '@mui/material';
+
+interface SeriesNameProps {
+  name: string;
+  labels?: Labels;
+  formattedName?: string;
+  isFormatted?: boolean;
+}
+
+/*
+ * Display a series with labels in bold and with a copy to clipboard feature if isFormatted is enabled
+ * Else it will only display the series in plain text (mostly use for performance reasons)
+ */
+export function SeriesName({ name, labels, formattedName, isFormatted }: SeriesNameProps) {
+  if (isFormatted && labels) {
+    return <FormatedSeriesName labels={labels} />;
+  }
+  return <span>{formattedName ?? name}</span>;
```

Comment:
If possible, this should still be computed from labels as the formattedName is slightly different from your computation

Check ``sum by (notexisitinglabel) (up)`` result with and without formatting ;)

Reply:
Strange not the same behavior as prometheus 🤔

Reply:
Hola @sjcobb I saw the PR #1085, any reason to not return `{}` as Prometheus when there is no label? 

I think it would be better to return the same result as Prometheus. And if user want to return the query, they can use the legend 😄 WDYT?